rsmtool/rsmexplain.py

```diff
@@ -65,6 +65,9 @@ def select_examples(featureset, range_size=None):
         selected_ids = fs_ids
     elif isinstance(range_size, int):
         selected_ids = shap.sample(fs_ids, range_size)
+    elif isinstance(range_size, tuple):
```

Comment:
suggestion: `range_size` could always be an array of ints, having 1, 2 or more corresponding to the different options.

Reply:
Yeah, I considered this but I thought it would be more confusing than the current set up. Let's wait to get some user feedback before making this change.